### PR TITLE
Adding fundamental models to the app.

### PR DESCRIPTION
**Pull request by mvandenbeuken**
Encryptor: Encrypts and decrypts a message with a given password.
Secret: Stores message content using an Encryptor. Has a secret Recipient.
Recipient: Belongs to a secret. Has an email and a phone number.

We now have a simple controller and form that submits a secret with an unpersisted password.

### CODE REVIEW

File: spec/features/secrets_spec.rb [hunk 1]
```diff
@@ -0,0 +1,117 @@
+describe "secret form" do
+  let(:required_field_error) { I18n.t "errors.messages.blank" }
```
Comment (rubene) [inline spec/features/secrets_spec.rb]: :+1:

**Comment by ryanjones** [comment]
Once we have a salt per Secret this is :elephant: :elephant: :elephant:

**Comment by rubene** [comment]
:elephant: :elephant: :elephant:

**Comment by ryanjones** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**